jobbergateapi2/utils.py

```diff
@@ -0,0 +1,40 @@
+"""
```

Comment:
Can you just rename this whole file `createsuperuser.py` since it doesn't get used for anything else?